src/js/live-tracker.js

```diff
@@ -13,6 +14,25 @@ class LiveTracker extends Component {
     this.reset_();
 
     this.on(this.player_, 'durationchange', this.handleDurationchange);
+
+    // we don't need to track live playback if the document is hidden,
+    // also, tracking when the document is hidden can
+    // cause the CPU to spike and eventually crash the page on IE11.
+    if ('hidden' in document && 'visibilityState' in document) {
+      this.on(document, 'visibilitychange', this.handleVisibilityChange);
+    }
+  }
+
+  handleVisibilityChange() {
+    if (this.player_.duration() !== Infinity) {
+      return;
+    }
+
+    if (document.hidden) {
+      this.stopTracking();
+    } else {
+      this.startTracking();
```

Comment:
does startTracking() do a check of when called? I think it does but wanted confirmation.

Reply:
Yeah it we are already tracking it will return right away.